lib/Runtime/Library/DateImplementation.h

```diff
@@ -385,28 +385,29 @@ namespace Js {
     const auto ConvertUInt32ToString_ZeroPad_4 = [](const uint32 value, char16 *const buffer, const CharCount charCapacity)
     {
         Assert(charCapacity >= 4);
+        uint16 value_uint16 = static_cast<uint16>(value);
```

Comment:
Not sure I see the value here. Should be the same logic whether 32 or 16 bit?

Reply:
I remember I put it there, because otherwise I need to cast `value + _u('0')` to 16 bit every time it's assigned to `buffer[]`, which takes char16.